### PR TITLE
Update prerequisites to reference GitHub platform-specific installers

### PR DESCRIPTION
**Pull request by jordanmccullough**
With Training Team Foundation classes focusing on a more thorough tour of the GHfW and GHfM apps, the prerequisites document now includes revised setup/installation instructions to exclusively point to https://windows.github.com and http://mac.github.com.

@github/training-team Can you take a look at this?

### CODE REVIEW

File: courses/_posts/2001-02-28-github-class-prerequisites.md [hunk 1]
```diff
@@ -29,21 +29,22 @@ We request that you use [version 1.7.9](https://github.com/git/git/blob/master/D
...
+### For _Mac OS X_ Machines
+Download the installer from <https://mac.github.com>. This provides a Mac OS X  graphical user interface and optional setup of Git and GitHub command line utilties.
+
+__Checking Git Installation__
+
+Your Mac may already have Git installed. To verify, follow these steps:
+
+1. The location of the installed Git can be found with the command `which git`.
 3. If `usr/bin/git`, it was likely installed by Apple or [Xcode](https://developer.apple.com/xcode/).
 4. If `usr/local/bin/git` it was likely installed by [Homebrew](http://mxcl.github.com/homebrew/).
```
Comment (matthewmccullough) [inline courses/_posts/2001-02-28-github-class-prerequisites.md:44]: or the GitHub for Mac app (same location)

**Comment by matthewmccullough** [comment]
You are a speed demon!  I was going to do this later today, but I can't say I'm sad you beat me to it. :smile:

File: courses/_posts/2001-02-28-github-class-prerequisites.md [hunk 2]
```diff
@@ -29,21 +29,22 @@ We request that you use [version 1.7.9](https://github.com/git/git/blob/master/D
...
-  <http://windows.github.com>
-2. Another option for advanced users is to use MSysGit:
-  <http://git-scm.com/download/win>
-
-### and for _Mac OS X_ Machines
-1. Install the command line edition of Git from the package available at:  
-  <http://git-scm.com/download/mac>
-2. The location of the installed Git can be found with the command `which git`.
+### For _Windows_ Machines
+Download the installer from <https://windows.github.com>. This provides the Git Shell running latest version of Git and a Windows graphical user interface.
```
Comment (matthewmccullough) [inline courses/_posts/2001-02-28-github-class-prerequisites.md:33]: Git Shell -> GitHub Shell

**Comment by matthewmccullough** [comment]
I added a few comments, @jordanmccullough, but with a quick eye from another training team member, I think we are good to :shipit:

**Comment by tlberglund** [comment]
NOTE: this tab is still open, and I intend to provide feedback :soon:

**Comment by tlberglund** [comment]
For some reason I thought this review would take time, and I put it off for several days. Turns out this was a quickie. Looks great! :shipit:

**Pull request merged**